linera-chain/Cargo.toml

```diff
@@ -12,7 +12,12 @@ repository.workspace = true
 version.workspace = true
 
 [features]
-test = ["tokio/macros", "linera-base/test", "linera-execution/test"]
+test = [
+    "tokio/macros",
+    "linera-base/test",
+    "linera-execution/test",
+    "linera-views/test",
```

Comment:
Why is that a problem if we don't use the testing library of `linera-views` ?

Reply:
Since we run `cargo check-all-features` in our CI, if I add a test only function to a trait for example on `linera-views`, when `cargo check-all-features` tries to check `linera-chain` compiled with the `test` feature, it will think that the test implementation of the function is missing in `linera-views`. That happens because `linera-chain`'s `test` feature doesn't include `linera-views`'s `test` feature, so the `test` implementation will be seen as `test` by `linera-views` but not by `linera-chain`, so CI will fail.
This happened here: https://github.com/linera-io/linera-protocol/pull/2607#discussion_r1801365566

So I figured we might as well fix this for all dependencies already